lib/jekyll/filters.rb

```diff
@@ -68,22 +68,33 @@ def slugify(input, mode = nil)
     end
 
     # Format a date in short format e.g. "27 Jan 2011".
+    # It allows to use ordinal format as well, in both UK
+    # (e.g. "27th Jan 2011") and US ("e.g. Jan 27th, 2011") formats.
+    # UK format is the default.
     #
     # date - the Time to format.
+    # type - if "ordinal" the returned String will be in ordinal format
+    # style - if "US" the returned String will be in US format.
+    #         Otherwise it will be in UK format.
     #
     # Returns the formatting String.
-    def date_to_string(date)
-      time(date).strftime("%d %b %Y")
+    def date_to_string(date, type = nil, style = nil)
+      generic_date_to_string(date, "b", type, style)
```

Comment:
- instead of `generic_date_to_string()` how about `stringify_date()` ?
- instead of passing `b` and `B` as parameters, how about passing `%b` and `%B` ?